inc/template-functions.php

```diff
@@ -113,6 +113,15 @@ function twentynineteen_can_show_post_thumbnail() {
 	return ! post_password_required() && ! is_attachment() && has_post_thumbnail();
 }
 
+/**
```

Comment:
@nielslange This doesn't seem like it should be included in this PR. Can you remove this function to sync back up with master?

Reply:
@kjellr Good catch! This code seemed to slip in there accidentally. I just removed it and updated this PR.